src/components/groups/group.middlewares.ts

```diff
@@ -33,6 +33,33 @@ export const leaveGroup = asyncWrapper(async (req: Request, res: Response, next:
   next()
 })
 
+export const isMemberInGroup = 
+asyncWrapper(async (req: Request, res: Response, next: NextFunction) => {
+  const kickableUser = await User.query()
+    .findOne({ id: parseInt(req.params.userid) })
+
+  if (!kickableUser) {
+    res.redirect('/not-found')
+  } else {
+    const usersInGroup = await Group.relatedQuery('users').for(req.group.id)
+    const inGroup = usersInGroup.find(user => user.id === kickableUser.id)
```

Comment:
Can we merge this to the first query, it would be much cleaner. Since we throw not found if the user not found and if the user not in the group either.